src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1082,8 +1082,8 @@ public void uninstall_noop(ChocolateyConfiguration config, Action<PackageResult>
                 }
                 else
                 {
-                    IPackage installedPackage = packageManager.LocalRepository.FindPackage(packageName);
-                    if (installedPackage != null) installedPackageVersions.Add(installedPackage);
+                    SemanticVersion semanticVersion = new SemanticVersion(config.Version);

```

Comment:
Not sure if this needs a guard, converting the string to semantic version may cause it to throw an error here. That may be fine though.